.travis.yml

```diff
@@ -20,21 +20,21 @@ cache:
     - lighthouse-viewer/node_modules
 install:
   - yarn
-  - npm run install-all
+  - yarn run install-all
 before_script:
   - gem install travis-artifacts
   - export DISPLAY=:99.0
   - export LIGHTHOUSE_CHROMIUM_PATH="$(pwd)/chrome-linux/chrome"
   - sh -e /etc/init.d/xvfb start
   - ./lighthouse-core/scripts/download-chrome.sh
-  - npm run build-all
+  - yarn run build-all
```

Comment:
Pretty sure the "run" is optional. 

`yarn blah` would be great for all the things.